pkg/agent/route/interfaces.go

```diff
@@ -80,4 +80,20 @@ type Interface interface {
 
 	// DeleteLocalAntreaFlexibleIPAMPodRule is used to delete related IP set entries when an AntreaFlexibleIPAM Pod is deleted.
 	DeleteLocalAntreaFlexibleIPAMPodRule(podAddresses []net.IP) error
+
+	// AddMulticlusterWireGuardRule is used to add IP rule for Multi-cluster WireGuard tunnel.
+	AddMulticlusterWireGuardRule(mark int, table int) error
```

Comment:
```suggestion
	AddIPRule(mark int, table int) error
```

Reply:
I didn't see anything special for Multicluster or Wireguard, I think just naming it as AddIPRule is ok.

Reply:
Done. Renamed to `AddIPRule`, as well as `DeleteIPRule`.